slices/libfontconfig1.yaml

```diff
@@ -0,0 +1,11 @@
+package: libfontconfig1
+
+slices:
+  libs:
+    essential:
+      - fontconfig-config_config
+      - libc6_libs
+      - libexpat1_libs
+      - libfreetype6_libs
+    contents:
+      /usr/lib/*-linux-*/libfontconfig.so.1*:
```

Comment:
Please include the copyright slice and its reference similary to what has been suggested in fontconfig-config slice.